package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "amp-github-apps",
+  "description": "A collection of GitHub Apps for the AMP Project",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "The AMP HTML authors",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ampproject/amp-github-apps.git"
+  },
+  "engines": {
+    "node": "^10.14.1"
+  },
```

Comment:
`amphtml` currently allows all LTS versions including 8. Maybe do so here too?

See [this snippet](https://github.com/ampproject/amphtml/blob/master/package.json#L7-L10).

Reply:
Since this is going to run on AppEngine, and AppEngine is currently running the latest LTS, I don't see a reason to have tests running on a lower LTS version